### DISCUSSION TITLE
How to Start developing for FBM API?

### DISCUSSION THREAD

**Discussion by j2318** [amzn/selling-partner-api-models#3454] [General] [answered]
Hi,
I am developer trying to create a private SP-API app.  Is it possible to test the API without any Amazon store?  I want to be able to connect to test (sandbox) store to see whether my calls are correct. Problem I am having is that it looks like I have to create an AWS account and Sign in to Seller Central using developer credentials.   Is it possible to work on SP-API without Seller account?
-- jp

**Comment by 0xs3n4** [comment]
I believe that you need aws account for access a SP-API services

**Comment by stefnats** [comment, marked as answer]
You will definitely need a paid Seller Central account and an AWS account (the free tier will suffice).

Then you can start using this guide:

https://developer-docs.amazon.com/sp-api/docs/registering-as-a-developer

The whole site gives you a lot of information, but it can be overwhelming.

**Comment by sungolivia** [comment]
@stefnats  is correct you would still need a Seller central account and AWS account - free tier should be enough!
